hw2/bhbruce/q1/main.cpp

```diff
@@ -0,0 +1,32 @@
+#include "line.hpp"
+#include <iostream>
+
+
+using namespace std;
```

Comment:
always avoid using this.

Reply:
Besides I used "std::" in the main function, what's the reason for your comment?

Thanks

Reply:
`using namespace std;`  means you can call STD functions without `std::`
i.g.
```
cout << "hello";
```

if you don't use  `using namespace std;` , then you need to write:
```
std::cout << "hello";
```

Why we don't like using namespace?
suppose that you have other namespace `myspace` and it has a function `cout`
then when you call `cout`, which `cout` is it? `std::cout` or `myspace::cout`